### PR TITLE
Add flow support and an example

### PR DESCRIPTION
**Pull request by nikgraf**

### CODE REVIEW

**Comment by mxstbr** [comment]
TODO: Test that flow extracts default props and state

File: client/components/Playground/ArrayControl/index.js [hunk 1]
```diff
@@ -35,8 +34,8 @@ const ArrayControl = (props) => {
         onClick={ () => onUpdate({ value: ArrayControl.randomValue(propTypeData) }) }
       />
         <div style={{ paddingLeft: 20 }}>
-          { (arrayPropTypeData.name === 'shape') ?
-            renderNestedArrayControls(arrayPropTypeData, rangeArray, value, onUpdateEntry) :
+          { (propTypeData.value.name === 'shape') ?
```
Comment (nikgraf) [inline client/components/Playground/ArrayControl/index.js]: this will not work … will fix it

Comment (nikgraf) [inline client/components/Playground/ArrayControl/index.js]: hmm maybe it actually does :)

Comment (mxstbr) [inline client/components/Playground/ArrayControl/index.js]: Why shouldn't it?

File: plugin/__test__/normalize-meta-info.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            value: '42',
+            computed: false,
+          },
+        ],
+      },
+    };
+    expect(normalizeMetaInfo(source)).to.deep.equal(expected);
+  });
+
+  it('should manage "type" in arrays', () => {
```
Comment (nikgraf) [inline plugin/__test__/normalize-meta-info.js]: that test doesn't do much ;)

File: client/components/Playground/ArrayControl/index.js (hunk 1 above)
Comment (nikgraf) [inline client/components/Playground/ArrayControl/index.js]: was just confused by the refactoring

**Pull request merged**